src/sentry/options/defaults.py

```diff
@@ -152,7 +152,6 @@
 register('snuba.search.max-chunk-size', default=2000)
 register('snuba.search.max-total-chunk-time-seconds', default=30.0)
 register('snuba.search.hits-sample-size', default=100)
-register('snuba.events-queries.enabled', type=Bool, default=True)
```

Comment:
This was stored on the DB to True. Could you please verify with matt if there is anything he has to do before we remove it ?

Reply:
@mattrobenolt Is there anything that needs to happen here? ☝️

Reply:
Confirmed with Matt that it's fine to merge before unsetting this. Can be removed either manually or via data migration later.